core/parentchain/light-client/src/lib.rs

```diff
@@ -454,15 +454,13 @@ impl<Block: BlockT> LightClientState<Block> for LightValidation<Block> {
 	}
 }
 
-pub fn grandpa_log<Block: BlockT>(
-	digest: &DigestG<HashFor<Block>>,
-) -> Option<ConsensusLog<NumberFor<Block>>> {
+pub fn grandpa_log<Block: BlockT>(digest: &DigestG) -> Option<ConsensusLog<NumberFor<Block>>> {
```

Comment:
Please rename the import to just `Digest`. The `G` stands for generic, which is not the case anymore.